experiment/nursery/main.go

```diff
@@ -142,3 +176,42 @@ func cacheHandler(cache *diskcache.Cache) http.Handler {
 		}
 	})
 }
+
+func monitorDiskAndEvict(cache *diskcache.Cache, interval time.Duration, minBlocksFree, minFilesFree float64) {
+	logger := log.WithFields(log.Fields{
+		"component": "nursery",
+	})
+	dir := cache.DiskRoot()
+	// forever check if usage is past thresholds and evict
+	for range time.Tick(interval) {
+		blocksFree, filesFree := diskutil.GetDiskUsage(dir)
+		// if we are past either threshold, evict until we are not
+		if blocksFree < minBlocksFree || filesFree < minFilesFree {
+			logger.WithFields(log.Fields{
```

Comment:
This might be useful to log every tick instead of only when we actually do evictions. Then we could monitor the disk usage more easily.

Reply:
SGTM